sys/compat/freebsd64/freebsd64_time.c

```diff
@@ -273,13 +273,17 @@ int
 freebsd64_ktimer_create(struct thread *td,
     struct freebsd64_ktimer_create_args *uap)
 {
-	struct sigevent_c ev, *evp;
+	struct sigevent64 ev64;
```

Comment:
This was an old bug and fixes all but 2 of the timer_create_test tests that used to fail.  I haven't yet debugged why two tests still fail.  I think they use `SIGEV_THREAD` and they work fine with CheriABI, just fail with freebsd64.